test/infrastructure/docker/Makefile

```diff
@@ -92,7 +92,7 @@ test-e2e: ## Run the end-to-end tests
 E2E_CONF_FILE ?= e2e/local-e2e.conf
 SKIP_RESOURCE_CLEANUP ?= false
 run-e2e:
-	go test ./e2e -v -ginkgo.v -ginkgo.trace -count=1 -timeout=20m -tags=e2e -e2e.config="$(abspath $(E2E_CONF_FILE))" -skip-resource-cleanup=$(SKIP_RESOURCE_CLEANUP)
+	go test ./e2e -v -ginkgo.v -ginkgo.trace -count=1 -timeout=35m -tags=e2e -e2e.config="$(abspath $(E2E_CONF_FILE))" -skip-resource-cleanup=$(SKIP_RESOURCE_CLEANUP)
```

Comment:
Sidenote: we keep increasing the timeout value 😬

Reply:
heh I noticed the same. But we're also adding more assertions that wait on Pod's running and their attributes

Reply:
Yeah I know. 😐sigh. 
But that's why I suggested opening an issue to run the tests in parallel! 🙂 

I can't think of any other way to reduce running time as we add more tests other than compounding all the behaviors under a single test. But at that point, we are writing an e2e test that says "test everything". I don't know the parameters of our CI but maybe we can tweak docker params to make it run faster if we start to see failures due to timeouts.

Reply:
Issue for running CAPD e2e in parallel https://github.com/kubernetes-sigs/cluster-api/issues/2795